cartoframes/data/enrichment/points_enrichment.py

```diff
@@ -0,0 +1,122 @@
+import uuid
+
+from ..clients import bigquery_client
+from ...auth import get_default_credentials
+from collections import defaultdict
+
+
+_ENRICHMENT_ID = 'enrichment_id'
+_WORKING_PROJECT = 'carto-do-customers'
+
+# TODO: process column name in metadata, remove spaces and points
+
+
+def enrich_points(data, variables, data_geom_column='geometry', filters=dict(), credentials=None):
+
+    credentials = credentials or get_default_credentials()
+    bq_client = bigquery_client.BigQueryClient(credentials)
+
+    user_workspace = credentials.username.replace('-', '_')
```

Comment:
A comment: get the user dataset name, should it be a responsibility of credentials class? If yes, I will add it to https://github.com/CartoDB/data-observatory/issues/147 issue

A detail: should we call it dataset? avoiding using several names for the same thing